packages/trilinoscouplings/examples/scaling/example_Maxwell.cpp

```diff
@@ -2523,6 +2551,7 @@ void TestMueLuMultiLevelPreconditioner_Maxwell(char ProblemType[],
                                                Epetra_CrsMatrix   & CurlCurl,
                                                Epetra_CrsMatrix   & D0clean,
                                                Epetra_CrsMatrix   & M0inv,
+                                               Epetra_CrsMatrix   & Ms,
```

Comment:
You can also pass Ms to RefMaxwell..

Reply:
Well that would be a good idea, now wouldn't it